py2pack/templates/opensuse-legacy.spec

```diff
@@ -23,7 +23,7 @@ Summary:        {{ summary_no_ending_dot|default(summary, true) }}
 Url:            {{ home_page }}
 Group:          Development/Languages/Python
 Source:         {{ source_url|replace(version, '%{version}') }}
-BuildRequires:  python-devel {%- if requires_python %} = {{ requires_python }} {% endif %}
+{%- if requires_python %} = {{ requires_python }} {% endif %}
```

Comment:
This is wrong. If `python_requires` is set in setuptools, it will break the .spec. Just remove the whole line for now.